sdk/scheduler/src/main/java/org/apache/mesos/api/PlansResource.java

```diff
@@ -193,4 +196,17 @@ public Response restartCommand(
             return Optional.empty();
         }
     }
+
+    static class CommandResultInfo {
```

Comment:
Is this new or moved from somewhere else?  Why the `Info` suffix?  To match protobufs?  But it isn't a protobuf.